spark/src/main/scala/geotrellis/spark/op/README.md

```diff
@@ -0,0 +1,158 @@
+# GeoTrellis Spark Raster Operations
+
+In `geotrellis.spark` we represent raster layers as a distributed collection of non-intersecting raster tiles indexed by keys from a given layout. For instance a spatial raster is represented as `RDD[(SpatialKey, Tile)]` where `SpatialKey(col: Int, row: Int)` from a `TileLayout`. In this setup we represent operations between raster layers as a join.
```

Comment:
I think this should read: "In `geotrellis.spark` we represent a distributed, non-intersecting collection of raster layers as tiles indexed by keys from a given layout." The mode of implementation seems to me orthogonal to the means of representation